oh.ps1

```diff
@@ -195,15 +196,18 @@ else {
 }
 
 # workaround to force 32bit JAVA in order to have DICOM working on 64bit arch
-if ( $DICOM_ENABLE -eq "on" ) {
-	Write-Host "DICOM_ENABLE=on, forcing JAVA architecture to 32bit" 
-	$script:JAVA_ARCH=32;
-	$script:JAVA_PACKAGE_ARCH="i686";
-	#$script:MYSQL_ARCH=32;
-}
+#
+# NOT NEEDED ANYMORE FROM OH 1.12-dev !
+#
 
-######## MySQL/MariaDB Software
-# MariaDB
+#if ( $DICOM_ENABLE -eq "on" ) {
```

Comment:
Is this still needed?  I thought when upgraded/updated the DICOM jars and native libs this wasn't needed any longer.

Reply:
ugh ... I didn't notice the comment characters at the beginning of the line 😦